newdle/client/src/locales/es/messages.js

```diff
@@ -0,0 +1 @@
"Logging in...","Made at <0>CERN</0>, the place where the web was born.":"Hecho en el <0>CERN</0>, el lugar donde naci\xF3 la web.","No options chosen":"No options chosen","No users matching the criteria":"No users matching the criteria","Note that some of your participants did not provide an email address and thus could not be notified!":"Note that some of your participants did not provide an email address and thus could not be notified!","Please enter your name...":"Please enter your name...","Select final date":"Select final date","Send your answer":"Send your answer","Set the time slots based on their availability":"Configura las franjas horarias seg\xFAn su disponibilidad","Some of your participants do not have e-mail addresses and will not be contacted:":"Some of your participants do not have e-mail addresses and will not be contacted:","Something when wrong when notifying participants:":"Something when wrong when notifying participants:","The following participants have not voted yet:":"The following participants have not voted yet:","The participants have been notified of the final date.":"The participants have been notified of the final date.","This newdle has already finished":"This newdle has already finished","This newdle has been deleted by its creator.":"This newdle has been deleted by its creator.","Update your answer":"Update your answer","Welcome to newdle!":"Bienvenido a newdle!","Who are you?":"Who are you?","You can use it to find out the best dates/times for your meetings.":"Puedes usarlo para conocer los mejores dias/fechas para tus reuniones.","Your answer has been saved!":"Your answer has been saved!","Your newdle was created and invitation e-mails have been sent. You can send the following link to everyone you would like to invite:":"Your newdle was created and invitation e-mails have been sent. You can send the following link to everyone you would like to invite:","Your newdle was created. You can now send the following to everyone you would like to invite:":"Your newdle was created. You can now send the following to everyone you would like to invite:","newdle is Open Source Software":"newdle es Software de c\xF3digo abierto","newdle logo":"newdle logo","originally created in the <0>{newdleTz}</0> timezone":["originally created in the <0>",["newdleTz"],"</0> timezone"],"switch back to the <0>{defaultUserTz}</0> timezone":["switch back to the <0>",["defaultUserTz"],"</0> timezone"],"{0, plural, one {# participant will be e-mailed:} other {# participants will be e-mailed:}}":[["0","plural",{one:["#"," participant will be e-mailed:"],other:["#"," participants will be e-mailed:"]}]],"{0} will take place on:":[["0"]," will take place on:"],"{numberOfTimeslots, plural, one {{numberOfAvailable} out of # option chosen} other {{numberOfAvailable} out of # options chosen}}":[["numberOfTimeslots","plural",{one:[["numberOfAvailable"]," out of ","#"," option chosen"],other:[["numberOfAvailable"]," out of ","#"," options chosen"]}]]};
```

Comment:
this file shouldn't be here. run `make` since you probably have an older version of lingui that creates it (newer versions use .mjs which is on gitignore)

Reply:
Removed.